Lib/multiprocessing/reduction.py

```diff
@@ -51,14 +51,35 @@ def dumps(cls, obj, protocol=None):
         cls(buf, protocol).dump(obj)
         return buf.getbuffer()
 
-    loads = pickle.loads
+    @classmethod
+    def loads(cls, bytes_object, *, fix_imports=True,
+                   encoding="ASCII", errors="strict"):
+        return pickle.loads(bytes_object, fix_imports=fix_imports,
+                            encoding=encoding, errors=errors)
+
+
+class AbstractPickler(ForkingPickler):
+    pass
+
+
+def register(type_, reduce_):
+    return ForkingPickler.register(type_, reduce_)
 
-register = ForkingPickler.register
 
 def dump(obj, file, protocol=None):
     '''Replacement for pickle.dump() using ForkingPickler.'''
     ForkingPickler(file, protocol).dump(obj)
 
+
+def dumps(obj, protocol=None):
```

Comment:
I think the `protocol` argument is never passed explicitly?

Reply:
This is so it matches the custom `dumps` in the AbstractPickler/ForkingPickler so when you stop using the custom context, it matches the methods (even if is not necessary because no other code uses it, it helps debugging if something fails).